.github/workflows/ci-build.yml

```diff
@@ -11,24 +11,18 @@ permissions:
   contents: read
 
 jobs:
-  build-cross:
+  build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - name: install dependencies
-        shell: bash
-        run: sudo apt-get install -y golang make
+      - name: checkout
+        uses: actions/checkout@v4
+      - name: setup go
+        uses: actions/setup-go@v5
+        with:
+          go-version: stable
+      - name: install taskfile
+        run: |
+          sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d -b /usr/local/bin
```

Comment:
Oh well this is not reproducible. 

Not sure if its a good idea to install stuff from random websites. Website can just disappear and break our CI.

Or they change the syntax and it becomes incompatible with our Taskfile

Dont they have a package in apt?

Reply:
We could switch to [GitHub action](https://taskfile.dev/installation/#github-actions)

Reply:
I guess with the github action it should be okay. Looks less sus to me.

Reply:
Fair enough